src/Components/Authentication/Types.ts

```diff
@@ -24,4 +24,11 @@ export interface FormProps {
   handleTypeChange?: (modalType: ModalType) => void
 }
 
+export interface ModalOptions {
+  copy?: string
+  signupIntent?: string
```

Comment:
@xtina-starr I've seen `signupReferer` roaming around too. Like here https://github.com/artsy/force/blob/master/src/desktop/components/auth_modal/view.coffee#L58

Should we keep it? or is it safe to remove

Reply:
Good catch. I'll add it.